src/config/settings.default.js

```diff
@@ -13,14 +8,13 @@ export default {
   MAX_POST_MESSAGE_LENGTH: 140,
 
   // Network settings.
-  API_BASE_URL:
-    process.env.APP_CONTEXT_ENV === "develop"
-      ? development.default.apiBaseUrl
-      : production.default.apiBaseUrl,
+  API_BASE_URL: process.env.API_BASE_URL
```

Comment:
Can you document this?

Reply:
I refactored it because I can't remember why I made it this weird, now it just uses the correct config file.